internal/indexer/moniker.go

```diff
@@ -136,10 +136,11 @@ func (i *Indexer) ensureImportMoniker(identifier string, packageInformationID ui
 
 // makeMonikerPackage returns the package prefix used to construct a unique moniker for the given object.
 // A full moniker has the form `{package prefix}:{identifier suffix}`.
-func makeMonikerPackage(obj types.Object) string {
+func makeMonikerPackage(obj ObjectLike) string {
 	var pkgName string
 	if v, ok := obj.(*types.PkgName); ok {
-		pkgName = strings.Trim(v.Name(), `"`)
+		// pkgName = strings.Trim(v.Name(), `"`)
+		pkgName = v.Imported().Path()
```

Comment:
Can I get an example of what this changes? (Also, delete the old commented code if you're confident in this new expression).

Reply:
Yup, added comment explaining.